Konnect August Release Notes (#4385)
Kong/docs.konghq.com: app/konnect/updates.md

from a single, cloud-based control plane, and provides a catalog of all deployed
services.

## August  2022

### 2022.08.31

**Generic OIDC**

: {{site.konnect_short_name}} launches generic OIDC support to allow Single-Sign-On (SSO) for platform login. Customers now have the ability to configure a separate OIDC-compliant Identity Provider (IdP) for {{site.konnect_short_name}}. Generic OIDC is an Enterprise feature.

**Personal Access Tokens**

: {{site.konnect_short_name}} users can now generate personal access tokens (PATs). PATs are used as an alternative method of authentication for [decK](/deck/1.14.x/guides/konnect/) commands avoiding the need to use traditional username and passwords.

**AWS Marketplace Listing**

> : {{site.konnect_short_name}} Enterprise can now be purchased through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-7zds3oxx3ntjy) or via private contract with your Amazon Web Services account team. This can greatly simplify the procuement process for existing AWS customers.

Reviewer: ```suggestion
: {{site.konnect_short_name}} Enterprise can now be purchased through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-7zds3oxx3ntjy) or via private contract with your Amazon Web Services account team. This greatly simplifies the procurement process for existing AWS customers.
```

**DevPortal Custom Domain Progress Indicator**

> : We've added an in interactive checklist when configuring a [custom domain](/konnect/dev-portal/customization/#custom-dev-portal-url) for your developer portal. Now you can get an indigation whether your DNS changes have propogated and when the SSL certificate has been generated by {{site.konnect_short_name}}.

Reviewer: ```suggestion
: We've added an in interactive checklist when configuring a [custom domain](/konnect/dev-portal/customization/#custom-dev-portal-url) for your developer portal. There is now an indicator to help understand if your DNS changes have propogated, and when the SSL certificate has been generated by {{site.konnect_short_name}}.
```

**Download option to all reports**

> : We have introduced an "Export" option to all reports inside the Overview page in Analytics. That will download an unpivoted CSV of the report to your local harddrive that can be used for offline analysis.

Reviewer: ```suggestion
: We have introduced an "Export" option to all reports inside the overview page in Analytics. Exporting a report downloads an unpivoted CSV to your local machine that can be used for offline analysis.
```
